content/Django tips and good practices - Bruno Michetti.md

```diff
@@ -0,0 +1,364 @@
+# Django: Tips and good practices
+
+**Abstract**: The Django framework save time and effort and help to create apps and REST APIs with maintainable code. This blog gives some practical examples of what to do and what not to do when you code in Python using Django.  
+
+**Summary**: This blog explains why good code structure and configuration is important in any environment. It describes Django’s main concepts and some easy ways to implement web app functionalities with existing tools that solve general problems. I also recommend good practices for efficiency and APIs and packages for session management and testing. Finally, I describe some things that, from my experience, are best to avoid. 
+
+## Start with good code
+Like all software developers, I want to create maintainable code. It’s not just for me but also for my partners. It feels good to know that the programs you use or maintain have good code quality and what you construct is easy to change, reuse, update, review, and so on.  
```

Comment:
I would use the word `build` instead of `construct`. The later sounds a lot weirder in this context than the former.